data/sql/updates/pending_db_world/rev_1553390628412350005.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1553390628412350005');
+
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (63414, 'spell_mimiron_spinning_up');
```

Comment:
- please always use `DELETE` before `INSERT`
- missing new line at the end of file